CHANGELOG.md

```diff
@@ -18,6 +18,7 @@ Versioning](https://semver.org/spec/v2.0.0.html).
     are returned.
   - If the `sensors` field is not provided, the software filters events from
     sensors owned by the customers the user belongs to.
+- Replaced `Vec<u32>` with `Vec<ID>` for `customer_ids`.
```

Comment:
We need to inform users of any changes that affect their experience. In particular, changes to GraphQL APIs must include information about which parts of the API have changed.

Reply:
done